lib/src/fnc/time.rs

```diff
@@ -67,53 +66,53 @@ pub fn group(_: &Context, mut args: Vec<Value>) -> Result<Value, Error> {
 	}
 }
 
-pub fn hour(_: &Context, mut args: Vec<Value>) -> Result<Value, Error> {
-	match args.len() {
-		0 => Ok(Utc::now().hour().into()),
-		_ => match args.remove(0) {
-			Value::Datetime(v) => Ok(v.hour().into()),
-			_ => Ok(Value::None),
-		},
-	}
+pub fn hour((datetime,): (Option<Value>,)) -> Result<Value, Error> {
+	let date = match datetime {
+		Some(Value::Datetime(v)) => v,
+		None => Datetime::default(),
+		Some(_) => return Ok(Value::None),
+	};
+
+	Ok(date.hour().into())
 }
 
-pub fn mins(_: &Context, mut args: Vec<Value>) -> Result<Value, Error> {
-	match args.len() {
-		0 => Ok(Utc::now().minute().into()),
-		_ => match args.remove(0) {
-			Value::Datetime(v) => Ok(v.minute().into()),
-			_ => Ok(Value::None),
-		},
-	}
+pub fn mins((datetime,): (Option<Value>,)) -> Result<Value, Error> {
+	let date = match datetime {
+		Some(Value::Datetime(v)) => v,
+		None => Datetime::default(),
+		Some(_) => return Ok(Value::None),
+	};
+
+	Ok(date.minute().into())
 }
 
-pub fn month(_: &Context, mut args: Vec<Value>) -> Result<Value, Error> {
-	match args.len() {
-		0 => Ok(Utc::now().day().into()),
-		_ => match args.remove(0) {
-			Value::Datetime(v) => Ok(v.day().into()),
-			_ => Ok(Value::None),
-		},
-	}
+pub fn month((datetime,): (Option<Value>,)) -> Result<Value, Error> {
+	let date = match datetime {
+		Some(Value::Datetime(v)) => v,
+		None => Datetime::default(),
+		Some(_) => return Ok(Value::None),
+	};
+
+	Ok(date.month().into())
```

Comment:
This is the fixed typo I mentioned. The `month` function used to accidentally call `day`